tutorials/hess.ipynb

```diff
@@ -72,12 +69,64 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data_store = DataStore.from_file(\n",
-    "    \"$GAMMAPY_DATA/hess-dl3-dr1/hess-dl3-dr3-with-background.fits.gz\"\n",
-    ")\n",
-    "mask = data_store.obs_table[\"TARGET_NAME\"] == \"Crab\"\n",
-    "obs_table = data_store.obs_table[mask]\n",
-    "observations = data_store.get_observations(obs_table[\"OBS_ID\"])"
+    "config = \"\"\"\n",
+    "general:\n",
+    "    logging:\n",
+    "        level: INFO\n",
+    "    outdir: .\n",
+    "\n",
+    "observations:\n",
+    "    datastore: $GAMMAPY_DATA/hess-dl3-dr1/hess-dl3-dr3-with-background.fits.gz\n",
+    "    filters:\n",
+    "        - filter_type: par_value\n",
+    "          value_param: Crab\n",
+    "          variable: TARGET_NAME\n",
+    "\n",
+    "reduction:\n",
+    "    dataset-type: MapDataset\n",
+    "    stack-datasets: true\n",
+    "    offset-max: 2.5 deg\n",
+    "    psf-kernel-radius: 0.3 deg\n",
+    "    geom:\n",
+    "        skydir: [83.633, 22.014]\n",
+    "        width: [5, 5]\n",
+    "        binsz: 0.02\n",
+    "        coordsys: CEL\n",
+    "        proj: TAN\n",
+    "        axes:\n",
+    "          - name: energy\n",
+    "            hi_bnd: 30\n",
+    "            lo_bnd: 1\n",
+    "            nbin: 5\n",
+    "            interp: log\n",
+    "            node_type: edges\n",
+    "            unit: TeV\n",
+    "    geom-irf:\n",
+    "        skydir: [83.633, 22.014]\n",
+    "        width: [5, 5]\n",
+    "        binsz: 0.2\n",
+    "        coordsys: CEL\n",
+    "        proj: TAN\n",
+    "        axes:\n",
+    "            - name: energy\n",
+    "              hi_bnd: 30\n",
+    "              lo_bnd: 1\n",
+    "              nbin: 11\n",
+    "              interp: log\n",
+    "              node_type: center\n",
+    "              unit: TeV\n",
+    "\n",
+    "model: model.yaml\n",
+    "\n",
+    "fit:\n",
+    "    fit_range:\n",
+    "        max: 100 TeV\n",
+    "        min: 1 TeV\n",
+    "\n",
+    "flux: {}\n",
+    "\"\"\"\n",
+    "filename = Path(\"config.yaml\")\n",
```

Comment:
Usually if one writes the config in Python scripts or Jupyter notebooks, one would just use it, not write it to file, no?
Wouldn't it be better to just pass it to some Config classmethod that takes a YAML config string and uses it?
Or if you want to write to file, suggest to use [%%writefile](https://ipython.readthedocs.io/en/stable/interactive/magics.html#cellmagic-writefile) and remove the Path import and code.

Reply:
Done